content/tutorial/01-simple-orders/01-order/03-sign-order/README.md

```diff
@@ -20,12 +20,18 @@ For signing, we will use the `UnsignedOrder` type from the `SDK`, along with the
 /// file: run.ts
 import type { Web3Provider } from '@ethersproject/providers';
 +++import { OrderBookApi, SupportedChainId, OrderQuoteRequest, OrderQuoteSideKindSell, OrderSigningUtils, UnsignedOrder } from '@cowprotocol/cow-sdk';+++
++++import { BigNumber } from 'ethers';+++
+
+
+    
 
 export async function run(provider: Web3Provider): Promise<unknown> {
   // ...
 
   const order: UnsignedOrder = {
     ...quote,
+    sellAmount: BigNumber.from(quote.sellAmount).add(BigNumber.from(quote.feeAmount)).toString(), // Quoted sellAmount must be added to quoted feeAmount
+    feeAmount: '0', // and feeAmount must be set to 0
```

Comment:
Maybe mention that the the fee is already priced in the buy amount.

Also, I would make a const `buyAmount` for readability